docs/guides/migration-from-redux.md

```diff
@@ -307,91 +307,108 @@ Changing logic from sagas to effects could be painful. Redux-saga allows us to w
 **Redux**
 
 ```js
-import { createStore, applyMiddleware } from 'redux'
+import { combineReducers, applyMiddleware, createStore } from 'redux'
 import createSagaMiddleware from 'redux-saga'
-import { put, takeEvery, delay } from 'redux-saga/effects'
+import { take, select } from 'redux-saga/effects';
+
+/**
+ * main code
+ */
+const INCREMENT = 'INCREMENT';
+const LOG_COUNTER = 'LOG_COUNTER';
+
+const counterReducer = (state = 0, action) => {
+  switch(action.type) {
+    case INCREMENT: return state + 1;
+    default: return state;
+  }
+}
+const counterSelector = state => state.counter
 
-const sagaMiddleware = createSagaMiddleware()
+function* logCounterSaga() {
+  while (true) {
+    yield take(LOG_COUNTER);
 
-const store = createStore(reducer, applyMiddleware(sagaMiddleware))
+    const counter = yield select(counterSelector);
 
-export function* incrementAsyncSaga() {
-  yield delay(1000)
-  yield put({ type: 'INCREMENT' })
+    console.log("counter: ", counter);
+  }
 }
 
-export function* watchIncrementAsyncSaga() {
-  yield takeEvery('INCREMENT_ASYNC', incrementAsyncSaga)
-}
+/**
+ * initialization code
+ */
+const sagaMiddleware = createSagaMiddleware()
+const store = createStore(
+  combineReducers({ counter: counterReducer }),
+  applyMiddleware(sagaMiddleware),
+)
+sagaMiddleware.run(logCounterSaga);
+
+/**
+ * dispatch
+ */
+store.dispatch({ type: LOG_COUNTER }); // 0
 
-sagaMiddleware.run(watchIncrementAsyncSaga)
+store.dispatch({ type: INCREMENT });
 
-store.dispatch({ type: 'INCREMENT_ASYNC' })
+store.dispatch({ type: LOG_COUNTER }); // 1
 ```
 
 **Reatom**
 
-Usage with explicit action type
+[![Live example of reatom and redux-saga](https://codesandbox.io/static/img/play-codesandbox.svg)](hhttps://codesandbox.io/s/reatom-redux-saga-5pq0e)
```

Comment:
Oh, I don't like these inserts. Very much the attention is taken away... And it seems that this is not to open the example, but to edit the text.

Reply:
![image](https://user-images.githubusercontent.com/1655916/67219118-2782c580-f430-11e9-9d66-71a1863866b5.png)

vs

![image](https://user-images.githubusercontent.com/1655916/67219152-36697800-f430-11e9-9544-1be8794572ac.png)